src/core/utils/misc.h

```diff
@@ -47,6 +47,22 @@ namespace Utils
 {
     namespace Misc
     {
+        // use Binary prefix standards from IEC 60027-2
+        // see http://en.wikipedia.org/wiki/Kilobyte
+        enum class SizeUnit
```

Comment:
By the way, since you began to use **enum class**, you can even put it outside Utils::Misc namespace (Utils child namespaces are intended for the functions - that was the idea). 
It will be fine. Just SizeUnit::Byte etc.